src/vsphere_cpi/lib/cloud/vsphere/cloud.rb

```diff
@@ -313,10 +304,10 @@ def create_vm(agent_id, stemcell_cid, vm_type, networks_spec, existing_disk_cids
             end
           end
 
-          if vm_type.key?('nsx') && !vm_type['nsx']['lbs'].nil?
-            security_groups = vm_type['nsx']['lbs'].map { |m| m['security_group'] }.uniq
+          unless vm_type.nsx_lbs.nil?
+            security_groups = vm_type.nsx_lbs.map { |m| m['security_group'] }.uniq
             security_groups.each { |sg| nsx.add_vm_to_security_group(sg, created_vm.mob_id) }
-            nsx.add_members_to_lbs(vm_type['nsx']['lbs'])
+            nsx.add_members_to_lbs(vm_type.nsx['lbs'])
```

Comment:
Should we also do `vm_type.nsx_lbs` here?

Reply:
Yes, I missed it